srcpkgs/lazarus-lcl-gtk2/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'lazarus-lcl-gtk2'
+pkgname=lazarus-lcl-gtk2
+version=1.8.4
+revision=1
+wrksrc=lazarus
+build_style=gnu-makefile
+makedepends="fpc gtk+-devel gdk-pixbuf-devel"
+depends="lazarus"
+short_desc="LCL units for gtk2"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="modified LGPL-2"
```

Comment:
Not an spdx short-identifier

Reply:
Fixed (it is a modified LGPL-2.0-or-later).